arviz/data/io_pystan.py

```diff
@@ -571,6 +571,7 @@ def get_sample_stats(fit):
         name = re.sub("__$", "", key)
         name = "diverging" if name == "divergent" else name
         data[name] = values
+    data.update({"max_treedepth": False})

```

Comment:
I don't think this is the correct thing to do.

I would not touch `io_` functions in schema PR.

Reply:
Ohh, should I make a separate PR for this? Please also suggest the correct way to add this variable.

Reply:
It should be a separate PR after we have discussed what variables are we interested in adding to `sample_stats`, there is no rush.

Reply:
Alright! Are there any more changes to be made in this naming convention?